test/spanningtrees/wilson.jl

```diff
@@ -0,0 +1,32 @@
+using Random
+
+@testset "Wilson" begin
+    gg = Grid([4,4])
+    for g in testgraphs(gg)
+        rt = @inferred(random_spanning_tree(g))
+        tree = SimpleDiGraph(rt.edges)
+
+        #Check that we do get a tree out
+        @test !is_cyclic(tree)
+        #Check that it is indeed spanning
+        @test nv(tree)==nv(g) && is_connected(tree)
+    end
+    
+    # What follows is a probabilistic test that checks that the algorithm samples from the correct distribution
+    # We use a result by Pemantle & Wilson that states that
+    # Uniform Spanning Trees are actually a Determinantal Point Process over the edges of the graph
+    # The kernel of the DPP can be computed explictly from the edge-incidence matrix
+    Random.seed!(1) #not sure if this is OK in test suite?
```

Comment:
IMO it's not required, RNG is seeded by a `default` value already. Any purpose of seeding with `1`?